pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "numpy",
     "qiskit >= 0.45.1, < 0.46",
     "qiskit-aer >= 0.13.1, < 0.14",
-    "iqm-client >= 15.2, < 16.0"
+    "iqm-client >= 16.0a1, < 17.0"
```

Comment:
It's possible that while the custom branch is still not merged into master, 2 or more new stable major versions of IQM Client are released (e.g. 16.0 and then 16.1); this range will allow qiskit-on-iqm to fail. Perhaps, it should be a strict dependency on `16.0a1` here, or put the range for `a`-packages only (e.g. `>= 16.0a1, <16.99a1`, but I'm not sure that works)

Reply:
Changed this to be explicit